wpilibj/src/main/java/edu/wpi/first/wpilibj/simulation/ElevatorSim.java

```diff
@@ -49,8 +51,8 @@ public ElevatorSim(
       double maxHeightMeters,
       boolean simulateGravity,
       double startingHeightMeters,
-      Matrix<N1, N1> measurementStdDevs) {
-    super(plant, measurementStdDevs);
+      double... measurementStdDevs) {
+    super(plant, RobotController.getBatteryVoltage(), measurementStdDevs);
```

Comment:
The problem with using the return value of RobotController.getBatteryVoltage() here is the battery voltage varies over time in simulation. You'd have to constantly update the max input, which doesn't seem reasonable.

Reply:
So should I set this to 12_V then?

Reply:
Imo, yes. The alternative isn't easily doable.